src/test/test_private_sts.c

```diff
@@ -4,7 +4,7 @@
 
 #ifdef STS_COMPILE_UNIT_TESTS
 
-#include "test.h"
```

Comment:
If you really want to test the file scoped functions in symtseries.c you could add a main() to the conditionally compiled test section (and this entire directory can go away).  You would then compile it as two separate targets: a shared lib and an executable (passing the test directive to the exe target)